framework/bin/d4j/d4j-info

```diff
@@ -75,12 +75,26 @@ my $PID = $cmd_opts{p};
 my $BID = $cmd_opts{b};
 
 my $project = Project::create_project($PID);
+my $project_dir = "$SCRIPT_DIR/projects/$PID";
+
 my $revision_id;
 my $revision_date;
 my $bug_report_id;
 my $bug_report_url;
+
 # Check version id
 if (defined $BID) {
+    if ( ! -e "${project_dir}/trigger_tests/${BID}" ) {
+        print("${PID}-${BID} is a non-existent bug\n");
+        exit 1;
+    }
+
+    my @bug_ids = $project->{_vcs}->get_bug_ids;
```

Comment:
Better to use `$project->get_bug_ids()` -- by default this will just delegate to the VCS subroutine, but it may be overridden for some projects.